kernel/data-pipeline/api/src/main/java/org/apache/shardingsphere/data/pipeline/api/check/consistency/DataConsistencyCheckResult.java

```diff
@@ -33,12 +31,33 @@ public final class DataConsistencyCheckResult {
     
     private final DataConsistencyContentCheckResult contentCheckResult;
     
+    private final DataConsistencyCheckIgnoredType checkIgnoredType;
+    
+    private final boolean ignored;
```

Comment:
1, It's better to put ignored related fields before check result fields;

2, `ignored` field may be not necessary, could we just add `isIgnored()` method;